docs/3-limit-order/2-limit-order-v2-endpoints.md

```diff
@@ -0,0 +1,208 @@
+
+# Create limit order
+Sends a POST request to the Jupiter Limit Order API to get the unsigned transactions needed to create an order.
+
+<details>
+  <summary>
+    <div>
+      <div className="api-method-box post">POST</div>
+      <p className="api-method-path">https://api.jup.ag/limit/v2/createOrder</p>
+    </div>
+  </summary>
+
+**Parameters in use in the below code example:**
+
+- `inputMint`: The mint address of the input token (required).
+- `outputMint`: The mint address of the output token (required).
+- `maker`: 
+- `payer`: 
+- `makingAmount`: 
+- `takingAmount`
+- `expiredAt`
+- `feeBps`
+- `computeUnitPrice`
+- `referral`
+- `inputTokenProgram`
+- `outputTokenProgram`
+- `wrapAndUnwrapSol`
+
+### Example response
+```typescript
+type CancelOrders = {
+  maker: string;
+
+  // "auto" sets the priority fee based on network congestion
+  // and it will be capped at 500,000
+  computeUnitPrice: string | "auto";
+
+  // Specific order account public keys to cancel/close
+  orders?: string[] | undefined;
+};
+
+type CancelOrdersResponse = {
+  txs: string[];
+};
+```
+</details>
+
+
+
+# Cancel limit order
+Sends a POST request to the Jupiter Limit Order API to get the unsigned transactions needed to cancel order(s).
+
+<details>
+  <summary>
+    <div>
+      <div className="api-method-box post">POST</div>
+      <p className="api-method-path">https://api.jup.ag/limit/v2/cancelOrders</p>
+    </div>
+  </summary>
+
+**Parameters in use in the below code example:**
+
+- `maker`: 
+- `computeUnitPrice`: Defaults to auto
+- `orders`: 
```

Comment:
```suggestion
- `maker`: The wallet address of the user associated to the order(s) that are being cancelled.
- `computeUnitPrice`: Used to determine a transaction's prioritization fee. Defaults to `auto`.
- `orders`: The public keys for orders that are intended to be cancelled. If not specified, this will generate the transaction(s) to close all orders associated to the `maker`.
```